client/analytics/components/report-table/index.js

```diff
@@ -83,6 +84,15 @@ ReportTable.propTypes = {
 	 * The endpoint to use in API calls.
 	 */
 	endpoint: PropTypes.string,
+	/**
+	 * Method names used to load more data for table items. If omitted, no call will
```

Comment:
Should this specify that these method names need to be available via `select( 'wc-api' )`?

Reply:
Makes sense! :+1: 

Done: 3d21724494c18af52e75729308e74af61458c96e.